doc/security/hardening-tool.rst

```diff
@@ -0,0 +1,44 @@
+.. _hardening:
+
+Hardening Tool
+##############
+
+Zephyr contains several optional features that makes the overall system
+more secure. As we take advantage of hardware features, many of these
+options are platform specific and besides it, some of them are not
+known by developers.
+
+To address this problem, we provide a tool that helps to check an
+application configuration option list against a list of hardening
+preferences defined by the **Security Group**. The tool is capable of
+understanding what is the build target and suggest specific
```

Comment:
The tool can identify the build target and based on that provides suggestions and recommendations on how to optimize the configuration for security..